Cargo.toml

```
@@ -12,11 +12,13 @@ spantrace = ["error-stack/spantrace"]
 eyre = ["error-stack/eyre"]
 serde = ["error-stack/serde"]
 hooks = ["error-stack/hooks"]
-
+grpc=[]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 error-stack = "0.4.1"
 thiserror = "1"
 tracing = "0.1"
+tonic = "0.10"
+tonic-types = "0.10"
```

This adjusts the dependencies to indicate they are optional.
https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

```suggestion
tonic = { version = "0.10", optional = true }
tonic-types = { version = "0.10", optional = true }
```